ceph_deploy/hosts/arch/uninstall.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from ceph_deploy.util.system import disable_service, stop_service
+from ceph_deploy.lib import remoto
+
+
+SYSTEMD_UNITS = [
+    'ceph-mds.target',
+    'ceph-mon.target',
+    'ceph-osd.target',
+    'ceph-radosgw.target',
+    'ceph-fuse.target',
+    'ceph-mgr.target',
+    'ceph-rbd-mirror.target',
+    'ceph.target',
+]
+
+
+def uninstall(distro, purge=False):
+    packages = [
+        'ceph',
+    ]
+
+    hostname = distro.conn.hostname
+    LOG = logging.getLogger(hostname)
+
+    # I need to stop and disable services prior package removal
```

Comment:
this reads like a packaging problem on Arch's side. It is the package itself that should have rules that take care of this. I guess this doesn't happen here because you have encountered this?

Reply:
Exactly, Arch packages don't start nor stop services, neither it enables them.

Reply:
Would you prefer to notify the user, instead of starting and enabling services?